### DISCUSSION TITLE
测试建议带上地区，个人用的香港，日本，新加坡

### DISCUSSION THREAD

**Discussion by wangzhilin777** [XIU2/CloudflareSpeedTest#492] [分享]
参数：-dn 20 -cfcolo HKG,NRT,FUK,OKA,KIX,SIN
之前跑出来最多3m/s，加上地区20m/s ，同时看一个YouTube的8k+一个4k流畅。延迟也很低。

**Comment by soleige** [comment]
-cfcolo 只能在httping下用 不太方便

**Reply by XIU2** [reply]
因为只有在通过 HTTP 方式访问 Cloudflare CDN 的情况下，才能从返回的头部数据中获得地区代码。
单纯 TCPing 是无法得知 IP 当前地区代码的。

总不能 TCPing 后，再去 HTTP 访问一次来获取地区代码吧，而且还要考虑丢包等情况下，访问 1 次正好丢了怎么办？访问次数太多时间就太长了，因此将其加到 HTTPing 中是非常恰当的，在 HTTPing 过程中顺便记录下来 colo 地区代码即可。
